virttest/utils_net.py

```diff
@@ -3462,15 +3462,19 @@ def get_ip_address_by_interface(ifname, ip_ver="ipv4", linklocal=False):
     :param ifname: interface name
     :param ip_ver: Host IP version, ipv4 or ipv6.
     :param linklocal: Whether ip address is local or remote
-    :raise NetError: When failed to fetch IP address (ioctl raised IOError.).
+    :raise NetError: When failed to fetch IP address.
     """
     if ip_ver == "ipv6":
         ver = netifaces.AF_INET6
         linklocal_prefix = "fe80"
     else:
         ver = netifaces.AF_INET
         linklocal_prefix = "169.254"
-    addr = netifaces.ifaddresses(ifname).get(ver)
+    try:
+        addr = netifaces.ifaddresses(ifname).get(ver)
+    except:
+        raise NetError(
```

Comment:
```suggestion
        raise IPAddrGetError(
```
I think we can raise the specific error type, such as `IPAddrGetError`.

Reply:
@YongxueHong I'm trying to look at the whole python file, I don't think it's a pure renaming issue, if we change it we'll add new `Class` and other code to support it. So please help review it again,  maybe keeping the current code is a good idea.

Reply:
It seems to me that the exception can be caught because the class is defined at the beginning and call in subsequent functions or classes. So if this name by changed, does it also need to create a class and call it when other function use it? Maybe I'm wrong, please help to point it out. Thanks a lot.
```
class NetError(Exception):

    def __init__(self, *args):
        Exception.__init__(self, *args)
```

Reply:
Hi @yanglei-rh 
Thanks for the explanation.
We raise the NetError here since somewhere will catch the `NetError`, right?
So we just focus make this function can raise such type `Exception` for the caller.
For the `IPAddrGetError` inherits to the `NetError`:
`class IPAddrSetError(NetError):`

So this update will not affect the current calling place, like:

Reply:
Hello @YongxueHong Thanks for you explanation, I'll update the name later. Thanks again.

Reply:
Hi @YongxueHong Updated, please help review it again, thanks a lot.